docs/versioning.md

```diff
@@ -0,0 +1,28 @@
+# SWF Versioning
+
+> :warning: **Note:** Packages are the only artifacts published from the UDS Software Factory repositories, as bundles are meant to be _created_, not consumed, by users.
```

Comment:
@Racer159 Agreed with the intent but was operating under the assumption we would probably do something similar to uds-core as far as dev/demo bundles from this repo. Essentially a baseline for to test something on top of SWF that doesn involve having to build all the things just a `uds deploy uds-swf-dev:0.1.0` or for growth `uds deploy uds-swf-demo`

Reply:
added some clarification on that point